repo/blob/s3/op

```diff
@@ -0,0 +1,201 @@
+2020/03/18 22:28:06 created session token with assume role: expiration: 2020-03-19 05:43:05 +0000 UTC
```

Comment:
Was this file accidentally included?

Reply:
oops yes. My bad. I don't need to update this PR right ? This file did not make it to the upstream PR .

Reply:
No need to update this PR.